gp_learner.py

```diff
@@ -415,6 +427,11 @@ def mutate_del_triple(child):
 
 
 def mutate_expand_node(child, pb_en_out_link):
+    """Expands a random node of the pattern, into a triple.
+    Randomly chooses an edge variable and a node variable for the triple.
```

Comment:
The new triple is anchored at the randomly selected node. Its other components (the leftover edge and other node) are new variables.